examples/minimal/package.json

```diff
@@ -11,7 +11,7 @@
         "@pulumi/pulumi": "^0.14.4-dev"
     },
     "devDependencies": {
-        "typescript": "^2.5.3"
+        "typescript": "^2.6.2"
```

Comment:
Nit, not related to this change, but if we're updating these, any reason not to go straight to `^3.0.0`?

Reply:
I wanted to move to the lowest version of typescript that supported the language feature we needed.  It didn't seem necessary to move more forward if we didn't need any new capabilities.  This also prevents us from unintentionally using any new language features that may not be available for some pulumi user that themselves it using an older TS version.